.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -11,7 +11,7 @@
 <!-- Before pull requests can be merged, they should provide: -->
 
 - A descriptive but concise pull request title
-- [Docstrings for all functions](https://github.com/numpy/numpy/blob/master/doc/example.py)
+- [Docstrings for all functions](https://github.com/numpy/numpy/blob/maintenance/1.3.x/doc/example.py)
```

Comment:
```suggestion
- [Docstrings for all functions](https://github.com/numpy/numpydoc/blob/main/doc/example.py)
```

Reply:
Good catch @pitkajuh! That was a super old link. I updated your PR to point to the current version of the file.